lib/submissions.ts

```diff
@@ -178,6 +178,23 @@ export const approveSubmission = async (
   return data;
 };
 
+/** mark an approved submission as panal approved passing in the id and approved by  */
+export const panelApproveSubmission = async (
+  submissionId: string,
+  approvedBy: string
+): Promise<Submission> => {
+  const { data } = await axios.patch(
+    `${ENDPOINT_API}/submissions/${submissionId}`,
+    {
+      editedBy: approvedBy,
+      submissionState: 'panelApproved',
```

Comment:
Could the string value be `panel-approved` instead. It would be nicer for the backend.

Reply:
sure